src/views/SignIn/SignIn.tsx

```diff
@@ -159,27 +159,28 @@ export class SignIn extends React.PureComponent<{}, any> {
     }
 
     resetPassword = () => {
-        alert
+        void alert
             .fire({
                 text: _("What is your username?"),
                 input: "text",
                 showCancelButton: true,
             })
-            .then((username) => {
-                post("/api/v0/reset", { username: username })
-                    .then((res) => {
-                        if (res.success) {
-                            void alert.fire(
-                                _("An email with your new password has been emailed to you."),
-                            );
-                        } else {
-                            console.error(res);
-                            errorAlerter(res);
-                        }
-                    })
-                    .catch(errorAlerter);
-            })
-            .catch(ignore);
+            .then(({ value: username }) => {
+                if (username) {
```

Comment:
same empty string feedback: previously we saw the server error ("username too short"), now the box closes as if the request had completed successfully.